tests/blas/nodes/dot_test.py

```diff
@@ -1,11 +1,73 @@
+#!/usr/bin/env python3
+
 import numpy as np
+
 import argparse
 import scipy
+import random
+
 import dace
 from dace.memlet import Memlet
 
 import dace.libraries.blas as blas
 
+import dace.libraries.blas.utility.fpga_helper as streaming
+from dace.libraries.blas.utility import memory_operations as memOps
+from dace.transformation.interstate import GPUTransformSDFG
```

Comment:
This seems leftover

Reply:
All but `import dace.libraries.blas.utility.fpga_helper as streaming` was removed